src/material/snack-bar/snack-bar-container.ts

```diff
@@ -247,8 +247,14 @@ export class MatSnackBarContainer extends BasePortalOutlet
     if (!this._announceTimeoutId) {
       this._ngZone.runOutsideAngular(() => {
         this._announceTimeoutId = setTimeout(() => {
-          const inertElement = this._elementRef.nativeElement.querySelector('[aria-hidden]');
-          const liveElement = this._elementRef.nativeElement.querySelector('[aria-live]');
+          const { nativeElement } = this._elementRef;
+          const inertElement = nativeElement.querySelector('.mat-snack-bar-aria-hidden');
+          const liveElement = nativeElement.querySelector('.mat-snack-bar-aria-live');
```

Comment:
You could remove the `querySelector` altogether by using a template reference and a `ViewChild` to get a hold of the element.